### ISSUE TITLE
Put geospatial lessons on hold?

### ISSUE THREAD

**Issue by maneesha** [datacarpentry/r-spatial-data-management-intro#33]
In issue #32 which started off just pointing out a broken link, @lwasser and @jsta noted these lessons need a lot of work... to the point of @lwasser saying she does not want to teach them until the bugs are fixed.  Can we put this lesson set on hiatus until everything is fixed?  If the developers/maintainers are not comfortable teaching from it I don't want to offer it to the general public.

**Comment by lwasser** [comment]
Hey maneesha. Talk to tracy. I've taught parts of them but this set needs an overhaul. Tons of redundancy in content. Tons of extra unneeded stuff for a 2 day workshop... The main geospatial lessons are fine. Tracy needs to make this call. We've had a hard time finding others w geospatial knowledge who'd want to contribute and I don't have time to do all of the development at this point.

I'm fine w pulling the link from the dc website just to these Intro lessons.

To be clear I didn't say I didn't want to teach them. I said they aren't ready for another instructor to teach them without more development.

Tracy should make this call.



Leah A. Wasser
---
Director, Earth Analytics Education Initiative
Earth Lab, CU Boulder
twitter: @leahawasser


________________________________

**Comment by jsta** [comment]
I agree. The main geospatial lessons linked from the DC lessons page are well polished and useful:

* [Working with raster data in R](http://neondataskills.org/tutorial-series/raster-data-series/)
* [Working with vector data in R](http://neondataskills.org/tutorial-series/vector-data-series/)

The [Introduction to Geospatial data](http://www.datacarpentry.org/r-spatial-data-management-intro/) seems like it ran out of steam before completion. I think it makes sense to remove this one from the DC lessons page pending further development.

**Comment by maneesha** [comment]
Thanks.  Even if the raster/vector lessons are complete and polished, I'm concerned about not having the intro section in place.  I brought this up at a staff meeting with @tracykteal and I think the consensus was to take them down entirely until lessosn for a complete workshop are in place.  We just need to put this in action.

**Comment by lwasser** [comment]
That's up to you guys. Truthfully you don't need to have the intro lessons to teach the workshop. They are unnecessary in my opinion because they are redundant to what is in the raster and vector lessons which are highly polished. That's how I've taught these lessons several times. Also the raster and vector lessons are more than 2 days of materials so having more is too much.

Have you looked closely at the raster and vector lessons? There is intro content throughout. I suggest just removing the Intro lessons and then you have an excellent 2-3 day spatial workshop which has been taught several times successfully. Again if you want to take them down it's a shame but that is your call!

Leah

Leah A. Wasser
---
Director, Earth Analytics Education Initiative
Earth Lab, CU Boulder
twitter: @leahawasser

**Comment by tracykteal** [comment]
Thanks all! I agree this is a great set of lessons, and I don't think we take them down. From this thread and other feedback, it sounds like we create a section on the DC lessons website for 'beta' or 'in development', so people know they're available, but not as tested. I'll put in a PR for that and link it here for your comments. 

We're just doing the Ecology lesson release and are coming back to the Genomics lessons now to clean those up. We're trying a way to have the community be able to be more involved in the Genomics lessons, as there have been similar challenges there in getting things across the finish line. Hopefully that works well, and we can use that same approach for Geospatial. After Genomics, Geospatial is the next priority. There's already a lot of great work here and enthusiasm about the topic!

**Comment by lwasser** [comment]
That sounds absolutely perfect Tracy!! I like that..maybe if people see them as in development we can get some traction around people working on them. Thanks!

Leah A. Wasser
---
Director, Earth Analytics Education Initiative
Earth Lab, CU Boulder
twitter: @leahawasser


________________________________

**Comment by tracykteal** [comment]
Also I want to be clear that Data Carpentry is committed to these lessons, including putting staff effort towards these lessons. It was a motion at the Data Carpentry in-person Steering Committee meeting. *Motion: A major focus of staff effort over over the next 6 months will be to finalize the existing Ecology, Genomics and Geospatial lessons.*

We're working in work cycles to work on these lessons. We're just finishing the Ecology release, and now starting on the Genomics update and release, and Geospatial will be the focus after that. We're working on creating more opportunities for community engagement for Genomics, that will hopefully work for Geospatial too!

**Comment by lwasser** [comment]
Thank you @tracykteal - that clarification is appreciated. a lot of time did go into the lessons and dropping them would be a shame. We just need to build community around them and i agree what @jsta that when we do `sf` is probably the way to go!

it's really nice and much faster than sp. i think by the time we build this community we'll be ready to embrace sf for everything. it works but mostly with baseplot which is ok. ggplot is just under dev but my guess is full support will be available soon!!!

**Comment by maneesha** [comment]
Thanks all.  I didn't mean for the lessons to be taken down entirely as they are too valuable for that.  I was just thinking of taking some time on hold to clean them up before they were offered to the general public.  I also took @lwasser 's comment about not feeling comfortable teaching from the intro lesson as applying to the entire lesson set.  I'm glad to hear that the other modules are in good shape.

**Comment by lwasser** [comment]
They are @maneesha ! I've actually taught the raster and vector lessons a bunch of times at this point - once in Oslo, Norway and several times here in the US!! The vector and raster lessons are in great condition and are fully ready to be taught. I've taught pieces of the intro lessons BUT the intro lessons are not complete.  i think they can be paired down to some very basic information! 
:)  

To clarify there are some changes in R where the community is moving towards a new package for vector data. @jsta was just mentioning that we should consider moving towards that if we work on the lessons again. I think building community first is the key to this effort but the lessons as they stand now are plenty current and ready to be taught!

**Comment by jsta** [comment]
It seems like a number of people are submitting instructor training checkout information to this repo. 

Is this appropriate place rather than https://github.com/datacarpentry/R-spatial-raster-vector-lesson? The latter set of lessons are much more polished. I am not clear on the distinction between the two repos.
